src/security/ServerOptions.cc

```diff
@@ -207,6 +209,43 @@ Security::ServerOptions::createStaticServerContext(AnyP::PortCfg &port)
 }
 
 void
+Security::ServerOptions::createSigningContexts(AnyP::PortCfg &port)
+{
+    const char *portType = AnyP::ProtocolType_str[port.transport.protocol];
+    if (!certs.empty()) {
+#if USE_OPENSSL
+        Security::KeyData &keys = certs.front();
+        Ssl::readCertChainAndPrivateKeyFromFiles(signingCert, signPkey, certsToChain, keys.certFile.c_str(), keys.privateKeyFile.c_str());
+#else
+        char buf[128];
+        fatalf("Missing OpenSSL support for %s_port %s", portType, port.s.toUrl(buf, sizeof(buf)));
```

Comment:
The `missing OpenSSL support for ...` phrasing can be easily misinterpreted as if Squid currently lacks code that would allow this port to work when Squid is built with OpenSSL. Or as if OpenSSL itself cannot support this configuration yet. I suggest rephrasing this error message as `--with-open-ssl required for ...` or something like that.

Reply:
done.